docling_core/types/doc/document.py

```diff
@@ -1738,112 +1736,35 @@ def print_element_tree(self):
             elif isinstance(item, DocItem):
                 print(" " * level, f"{ix}: {item.label.value}")
 
-    def export_to_element_tree(self) -> str:
-        """Export_to_element_tree."""
-        texts = []
-        for ix, (item, level) in enumerate(self.iterate_items(with_groups=True)):
-            if isinstance(item, GroupItem):
-                texts.append(
-                    " " * level + f"{ix}: {item.label.value} with name={item.name}"
-                )
-            elif isinstance(item, DocItem):
-                texts.append(" " * level + f"{ix}: {item.label.value}")
-
-        return "\n".join(texts)
-
-    def save_as_json(
-        self,
-        filename: Path,
-        artifacts_dir: Optional[Path] = None,
-        image_mode: ImageRefMode = ImageRefMode.EMBEDDED,
-        indent: int = 2,
-    ):
-        """Save as json."""
-        artifacts_dir, reference_path = self._get_output_paths(filename, artifacts_dir)
-
-        if image_mode == ImageRefMode.REFERENCED:
-            os.makedirs(artifacts_dir, exist_ok=True)
-
-        new_doc = self._make_copy_with_refmode(
-            artifacts_dir, image_mode, reference_path=reference_path
-        )
+    def export_to_dict(self) -> Dict:
+        """export_to_dict."""
+        return self.model_dump(mode="json", by_alias=True, exclude_none=True)
 
-        out = new_doc.export_to_dict()
-        with open(filename, "w") as fw:
-            json.dump(out, fw, indent=indent)
-
-    def save_as_yaml(
-        self,
-        filename: Path,
-        artifacts_dir: Optional[Path] = None,
-        image_mode: ImageRefMode = ImageRefMode.EMBEDDED,
-        default_flow_style: bool = False,
-    ):
-        """Save as yaml."""
-        artifacts_dir, reference_path = self._get_output_paths(filename, artifacts_dir)
-
-        if image_mode == ImageRefMode.REFERENCED:
-            os.makedirs(artifacts_dir, exist_ok=True)
-
-        new_doc = self._make_copy_with_refmode(
-            artifacts_dir, image_mode, reference_path=reference_path
-        )
-
-        out = new_doc.export_to_dict()
-        with open(filename, "w") as fw:
-            yaml.dump(out, fw, default_flow_style=default_flow_style)
-
-    def export_to_dict(
```

Comment:
we should not remove any export_to_... methods

Reply:
I apologize for this, definitely not intentional, git has a way of making me do foolish things. Working to restore this code to this branch.